servicekeeper-configsource/servicekeeper-configsource-common/src/main/java/io/esastack/servicekeeper/configsource/cache/ConfigCacheImp.java

```diff
@@ -52,12 +52,22 @@ public ExternalConfig configOf(ResourceId resourceId) {
             final ArgResourceId matchAllId = new ArgResourceId(argId.getMethodId(),
                     argId.getArgName(), VALUE_MATCH_ALL);
 
+            config = configs.get(matchAllId);
+            if (config == null) {
+                // fallback to get arg config which name is argId.getMethodAndArgId()
+                config = configs.get(argId.getMethodAndArgId());
+            }
+
             if (logger.isDebugEnabled()) {
-                logger.debug("Obtained {}'s config: {}", matchAllId, configs.get(matchAllId));
+                logger.debug("Obtained {}'s config: {}", resourceId, config);
```

Comment:
Please add more logs to describe where the current config got from? directly by resourceId or fallback to arg value which can apply to all values?